PhysicsTools/PythonAnalysis/test/imports.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -x
+set -e
+
+echo ">>> Create temporary directory for cache"
+TEST_TMPDIR=$(mktemp -d /tmp/cmssw_theano.XXXXXXX)
```

Comment:
why not use $CMSSW_BASE/tmp/cmssw_theano.XXXXXXX

Reply:
Git grep showed that almost nothing uses it. It's a bit sad that according to documentation `mktemp` does not look into `TMPDIR`. If you want people to use `$CMSSW_BASE/tmp` as temporary directory/scratch place we should set `TMPDIR` within CMSSW to `$CMSSW_BASE/tmp`.

We can change it if you want.